front/src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
```

Comment:
we could check if these classes are being used anywhere. If not, we can remove them